lib/lyber_core/robot.rb

```diff
@@ -12,8 +12,8 @@ class Robot
     sidekiq_retries_exhausted do |job, ex|
       # When all the retries are exhausted, update the workflow to error.
       robot = job['class'].constantize.new
-      workflow = Workflow.new(workflow_service: WorkflowClientFactory.build,
-                              druid: job['args'].first,
+      druid = job['args'].first
+      workflow = Workflow.new(object_client: Dor::Services::Client.object(druid),
```

Comment:
andbutalso, I noticed that `workflow` is both a local variable here and a private method below. Intended?

Reply:
i'm inclined to leave this one as-is also, but similarly happy to circle back if you end up wanting to revisit